pkg/kratos/service.go

```diff
@@ -387,6 +393,78 @@ func (s *Service) UpdateRecoveryFlow(
 	return returnToResp, resp.Cookies(), nil
 }
 
+func (s *Service) executeIdentifierFirstUpdateLoginRequest(
```

Comment:
thought: this looks like logic that should be part of the internal kratos client, not service logic